controllers/gatekeeper_controller.go

```diff
@@ -1102,21 +1102,19 @@ func openShiftDeploymentOverrides(obj *unstructured.Unstructured) error {
 }
 
 func setMutationFlags(obj *unstructured.Unstructured, webhookConfig *operatorv1alpha1.WebhookConfig) error {
-	if webhookConfig == nil {
-		return nil
-	}
-
-	if webhookConfig.LogMutations != nil && webhookConfig.LogMutations.ToBool() {
+	if webhookConfig != nil && webhookConfig.LogMutations != nil && webhookConfig.LogMutations.ToBool() {
```

Comment:
I think the `if webhookConfig == nil {` block can be restored and the check removed elsewhere with the new changes?